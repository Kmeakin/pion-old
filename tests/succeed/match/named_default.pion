//~ run = "${PION} elab ${FILE}"

fun (x : Int) => match x {
    0 => 1,
    otherwise => otherwise,
}
