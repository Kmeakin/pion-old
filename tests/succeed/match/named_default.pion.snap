exit status: 0
stdout:
fun (x : Int) => match x {0 => 1, otherwise => otherwise} : Int -> Int

stderr:
